gherkin/ruby/lib/gherkin/id_generator.rb

```diff
@@ -0,0 +1,35 @@
+require 'securerandom'
+
+module Gherkin
+  class IdGenerator
+    @@use_incrementing ||= false
+
+    def self.new_id()
```

Comment:
Please avoid class methods. We should use two distinct implementations of the `IdGenerator` contract, which is a class with a `new_id` instance method.